node/src/main/scala/com/wavesplatform/transaction/smart/InvokeScriptTransaction.scala

```diff
@@ -65,7 +65,7 @@ object InvokeScriptTransaction extends TransactionParser {
   override def parseBytes(bytes: Array[Byte]): Try[InvokeScriptTransaction] =
     serializer.parseBytes(bytes)
 
-  case class Payment(amount: TxAmount, assetId: Asset)
+  case class Payment(amount: Long, assetId: Asset)
```

Comment:
Positive? https://github.com/wavesplatform/Waves/blob/version-1.3.x/node/src/main/scala/com/wavesplatform/transaction/validation/impl/InvokeScriptTxValidator.scala#L21

Reply:
I think it may be problem here for transactions before `forbidSyncDAppNegativePaymentHeight `:
https://github.com/wavesplatform/Waves/blob/version-1.3.x/node/src/main/scala/com/wavesplatform/state/diffs/invoke/InvokeScriptDiff.scala#L82